py3status/modules/wwan_status.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+Display current network and ip address for newer Huwei modems. It
```

Comment:
We miss a one line docstring for the short description please (see the other modules for example)